### PR TITLE
Update Nginx SDK to API version 2023 04 01

### PR DESCRIPTION
**Pull request by SpencerOfwiti**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Review by jsquire — DISMISSED** [review]
@SpencerOfwiti:

Our automation is identifying you as a community contributor.  Before moving forward with this PR, please review the Azure SDK onboarding documentation and ensure that your GitHub organization membership and permissions are correct.
- [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
- [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_

Please also be sure to add yourself to [CODEOWNERS](https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS) for this library, if you will be maintaining it going forward.

**Comment by SpencerOfwiti** [comment]
@jsquire I have made changes.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -645,6 +645,9 @@
 # ServiceLabel: %Network - Private Link %Service Attention
 #/<NotInRepo>/                                                     @privlinksuppgithub
 
+# PRLabel: %Nginx
+/sdk/nginx/														   @sofwiti @limgu @yashi
```
Comment (jsquire) [inline .github/CODEOWNERS:649]: None of these folks are eligible to be CODEOWNERS, as they do not have write access to the repository.   Please follow the Azure SDK onboarding guide and ensure that they've requested write access, have joined the Microsoft and Azure orgs, and those memberships are flagged as public.

**Comment by jsquire** [comment]
> @jsquire I have made changes.

Your organization memberships need to be marked as public, which they are not currently.   Please also ensure that you've requested write access to the repository, as described in the Azure SDK onboarding guide.

File: .github/CODEOWNERS (hunk 1 above)
Comment (SpencerOfwiti) [reply .github/CODEOWNERS]: I have updated the CODEOWNERS with the right GitHub accounts

**Comment by SpencerOfwiti** [comment]
> > @jsquire I have made changes.
> 
> Your organization memberships need to be marked as public, which they are not currently. Please also ensure that you've requested write access to the repository, as described in the Azure SDK onboarding guide.

@jsquire I have made my membership public and gotten write permissions.

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**